README.md

```diff
@@ -14,6 +14,11 @@ ember install ember-cli-blanket
 ember install:addon ember-cli-blanket
 ```
 
+## Requirements
+* If using Mocha, Testem `>= 1.6.0` for which you need ember-cli `> 2.4.3`
```

Comment:
Yeah, we're on mocha across the board - I'll take a peek at the history for this in `ember-cli-code-coverage` and see if there is an attached reason